pkgs/by-name/st/stract/package.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  openssl,
+  curl,
+  fetchurl,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "stract";
+  version = "0-unstable-2024-09-14";
+
+  src = fetchFromGitHub {
+    owner = "StractOrg";
+    repo = "stract";
+    rev = "21d28ea86d9ff19ccb4c25b1bdde67e5ec302d79";
+    hash = "sha256-7Uvo5+saxwTMQjfDliyOYC6j6LbpMf/FiONfX38xepI=";
+  };
+
+  cargoHash = "sha256-7Skbeeev/xBAhlcyOsYpDJB9LnZpT66D0Fu1I/jIBso=";
+
+  cargoDepsName = pname;
```

```suggestion
  cargoDepsName = stract;
```
Don't interpolate `pname`.

---

Hm, why not?

---

Overriding `pname` should not alter the build output, and it is considered a best practice to not use it superfluously. See #277994.